lib/timeutils/scan-timestamp.c

```diff
@@ -244,7 +244,10 @@ __is_bsd_linksys(const guchar *src, guint32 left)
 {
   /* "MMM DD HH:MM:SS YYYY " */
   return (left >= 21
-          && __is_bsd_rfc_3164(src, left)
+          && src[3] == ' '
+          && src[6] == ' '
```

Comment:
Could you elaborate more what is happening here? I do not really understand the commit message. What I can see is a function was inlined and eliminated.

Reply:
The function was not eliminated as it is retained for BSD log format detection. Here's my rationale in more details:

* although the BSD and LinkSys formats are the same in their initial characters, the *reason* for potentially changing either logic is separate. e.g. either can change without the other having to change.
  - I might want to improve the BSD one (which I do in a followup patch), which I explicitly don't want to get applied to the LinkSys one, because I know that linksys generates formats in a specific way.
  - by separating these the code gets longer (so DRY would apply), but at the same time the two formats are conceptually different, so it is not a good idea to reuse the logic